app/uk/gov/hmrc/eusubsidycompliancefrontend/persistence/VerifiedEoriCache.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eusubsidycompliancefrontend.persistence
+
+import cats.implicits.toFunctorOps
+import org.mongodb.scala.model._
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.VerifiedEori
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.types.EORI
+import uk.gov.hmrc.mongo.MongoComponent
+import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats.Implicits.jatLocalDateFormat
+import uk.gov.hmrc.mongo.play.json.{Codecs, PlayMongoRepository}
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.duration.HOURS
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class VerifiedEoriCache @Inject() (
+  mongoComponent: MongoComponent
+)(implicit ec: ExecutionContext)
+    extends PlayMongoRepository[VerifiedEori](
```

Comment:
I think the same could be achieved with `MongoCacheRepository` with less boilerplate code.

Reply:
Discussed on a call and decided there's not a huge improvement, but could be reconsidered later.